src/org/opensolaris/opengrok/analysis/AnalyzerGuru.java

```diff
@@ -531,6 +531,9 @@ public static void writeXref(FileAnalyzerFactory factory, Reader in,
         args.setProject(project);
 
         FileAnalyzer analyzer = factory.getAnalyzer();
+        RuntimeEnvironment env = RuntimeEnvironment.getInstance();
```

Comment:
While testing the new highlighter, I found that scopes were not working when looking at historical versions of files or when annotating a file — i.e. where this `writeXref()` is called by `list.jsp` instead of its using the stored .gz version.